Restoring a task yields `Could not restore calendar or event`

with this console error:
```

[ERROR] calendar: could not restore /nextcloud/remote.php/dav/calendars/admin/trashbin/objects/2898.ics 
Object { app: "calendar", uid: "admin", error: TypeError }
ConsoleLogger.js:41:16
    log ConsoleLogger.js:41
    error ConsoleLogger.js:63
    e Trashbin.vue:225
    A runtime.js:63
    _invoke runtime.js:293
    w runtime.js:118
    h calendar-main.js:2
    s calendar-main.js:2
    (Async: promise callback)
    h calendar-main.js:2
    i calendar-main.js:2
    m calendar-main.js:2
    m calendar-main.js:2
    restore Trashbin.vue:173
    click Trashbin.vue:1
    VueJS 63

```

--- a/src/store/calendars.js
+++ b/src/store/calendars.js
@@ -706,8 +706,8 @@ const actions = {
 		const component = calendarObject.calendarComponent.getFirstComponent(vobject.objectType)
 		const timeRange = getters.getTimeRangeForCalendarCoveringRange(
 			vobject.calendar.id,
-			component.startDate.unixTime,
-			component.endDate.unixTime,
+			component.startDate?.unixTime,
+			component.endDate?.unixTime,
 		)
 		if (timeRange) {
 			commit('deleteFetchedTimeRangeFromCalendar', {

